### DISCUSSION TITLE
GitHub Workflow uses an old version of `libyaml` package

### DISCUSSION THREAD

**Discussion by WaKeMaTTa** [community/community#29639] [Actions]
Hello,

I have a [script](https://gist.github.com/WaKeMaTTa/f854a210bec68a2f524d052f9dc28083) that uses `pysch` ruby gem to well format YAML files.

When I execute this script in GitHub Workflow. I get the following error:

```
Your libyaml library is outdated: please upgrade to >= 0.2.5.
```

The solution is to run re-install the gem using `--enable-bundled-libyaml` compilation flag:

```bash
# github workflow step:
- name: "Re-install psych (--enable-bundled-libyaml)"
  run: gem install psych -v '4.0.3' -- --enable-bundled-libyaml
```

but github workflow throws me this error:

```
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /var/lib/gems/3.1.0 directory.
```

**Solution**

Please upgrade the libyaml version to v0.2.5

**Comment by airtower-luna** [comment]
That seems like a request for the https://github.com/actions/runner-images repository. :wink: 

A quick fix could be to use a job container with a newer libyaml version, e.g. Debian testing/unstable (but mind the speaking name :sweat_smile:).

**Reply by WaKeMaTTa** [reply]
Thanks. I will post it there.
